pkg/network/apis/network/validation/validation.go

```diff
@@ -121,6 +126,13 @@ func ValidateNetNamespace(netnamespace *networkapi.NetNamespace) field.ErrorList
 	if err := network.ValidVNID(netnamespace.NetID); err != nil {
 		allErrs = append(allErrs, field.Invalid(field.NewPath("netid"), netnamespace.NetID, err.Error()))
 	}
+
+	for i, ip := range netnamespace.EgressIPs {
+		if net.ParseIP(ip) == nil {
+			allErrs = append(allErrs, field.Invalid(field.NewPath("egressIPs").Index(i), ip, "invalid IP address"))
```

Comment:
Is this allowed to be localhost?  Multicast?  Any implications of those? (extra validation is needed only if it there is no clear use case for ever allowing those)

Reply:
There's no use case for setting it to either of those, but there's no danger either. (It would just fail, just like if they tried to claim a random far-away IP address; EgressIPs have to be on the same subnet as some node's primary IP.) We don't check for semantically bad IPs in validation anywhere else, so I think just letting it through here and then failing later makes sense.